frontend/src/core/comments/components/Comment.js

```diff
@@ -14,16 +15,25 @@ import type { TranslationComment } from 'core/api';
 
 type Props = {|
     comment: TranslationComment,
+    canPin?: boolean,
+    updatePinnedComment?: (boolean, number) => void,
 |};
 
 
 export default function Comment(props: Props) {
-    const { comment } = props;
+    const { comment, canPin, updatePinnedComment } = props;
 
     if (!comment) {
         return null;
     }
 
+    const handlePinnedComment = () => { 
```

Comment:
We should either have two separate methods or change the name to something like `handlePinAndUnpin`.